### DISCUSSION TITLE
Checkbox in table

### DISCUSSION THREAD

**Discussion by rtritto** [chakra-ui/zag#1605] [Q&A] [answered]
How to implement checkbox in a table?

Example:

<img width="665" alt="image" src="https://github.com/chakra-ui/zag/assets/40242971/f3fc7179-6076-48a2-8cb4-39e49efd94ce">

Is a new component (feature) needed?

**Comment by anubra266** [comment]
@rtritto Your question is a little vague, could you give more infor on what you're trying to achieve. Thanks

**Reply by rtritto** [reply]
I need to select multiple rows on a table. How can I do it using _zag_?

**Reply by anubra266** [reply, marked as answer]
There's no table machine in zag. You'd need a library for table data like [Tanstack table](https://tanstack.com/table/latest)

**Reply by rtritto** [reply]
Thanks. It will be done in the future?

**Reply by segunadebayo** [reply]
No, we don't intend to model a table component in the future. The Tanstack table works just fine.
